libzhlgen/test/functions/ELF_amd64/1.6.13/ShipSystem.zhl

```diff
@@ -98,6 +98,12 @@ cleanup __amd64 void ShipSystem::constructor(ShipSystem *this, int systemId, int
 "55534889fbbf????????4883ec18f30f1144240c":
 cleanup __amd64 bool ShipSystem::PartialDamage(ShipSystem *this, float amount); /* Thunk of ShipSystem::DamageOverTime, maybe it can be used instead? */
 
+"4157b840":
```

Comment:
These new signatures should ideally be relative to speed up the 64-bit scan.
Also have not validated yet.